playground/PopoverFrame.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useCallback, useState} from 'react';
+import {Frame, Button, Popover} from '@shopify/polaris';
+
+export function PopoverFrame() {
+  const [popoverActive, setPopoverActive] = useState(true);
+
+  const togglePopoverActive = useCallback(
+    () => setPopoverActive((popoverActive) => !popoverActive),
+    [],
+  );
+
+  const activator = (
+    <Button onClick={togglePopoverActive} disclosure>
+      More actions
+    </Button>
+  );
+
+  return (
+    // Frame without topBar doesn't have the issue.
```

Comment:
I can still replicate the issue both in `Playground` in a `Frame` without the `TopBar`, and in online-store-web after manually removing the `data-polaris-top-bar` attribute from the `TopBar` div.

https://user-images.githubusercontent.com/17032120/154038886-744640c4-5004-434f-8414-5bc435eed59e.mov

It seems to be related to whether the activator is on screen? Going to dig further.